docs/manifest.md

```diff
@@ -0,0 +1,1489 @@
+```
+$ sudo osbuild --export tree --output-directory output/1 example-1.json
+```
+Notice that we used `sudo` to run osbuild now. When generating a tree, osbuild must be run as root. Superuser privileges are required for some of osbuild's inner workings and for certain stages.
+
+The `--export` option tells osbuild which pipeline to export. Most useful manifests define multiple pipelines, many of which are used as intermediate steps in the build process. Usually, we only need to export one pipeline, the last one, but it's also sometimes useful to export multiple pipelines. In those cases, the `--export` option can be specified multiple times.
+
+The `--output-directory` option should be self-explanatory. The result of each pipeline listed in the `--export` options will be placed in this directory under a subdirectory with the pipeline's name.
+
+The output from the command should be:
+```
+starting example-1.json
+Pipeline example: fa0e466784d49682a1b7b3cb129b7a75b16bff9c2aed6aee3f0f1988056ce85a
+Build
+  root: <host>
+  runner: org.osbuild.fedora38 (org.osbuild.fedora38)
+org.osbuild.truncate: cdf6bd2e0d305beac977095697a8ac0cec4d6f744847363081dfb6ad62c389f3 {
+  "filename": "/newfile",
+  "size": "1G"
+}
+/usr/lib/tmpfiles.d/abrt.conf:2: Failed to resolve user 'abrt': No such process
+/usr/lib/tmpfiles.d/abrt.conf:9: Failed to resolve user 'abrt': No such process
+/usr/lib/tmpfiles.d/gluster.conf:2: Failed to resolve user 'gluster': No such process
+/usr/lib/tmpfiles.d/screen.conf:2: Failed to resolve group 'screen': No such process
+Failed to open file "/sys/fs/selinux/checkreqprot": Read-only file system
+
+⏱  Duration: 0s
+org.osbuild.chmod: fa0e466784d49682a1b7b3cb129b7a75b16bff9c2aed6aee3f0f1988056ce85a {
+  "items": {
+    "/newfile": {
+      "mode": "0444"
+    }
+  }
+}
+/usr/lib/tmpfiles.d/abrt.conf:2: Failed to resolve user 'abrt': No such process
+/usr/lib/tmpfiles.d/abrt.conf:9: Failed to resolve user 'abrt': No such process
+/usr/lib/tmpfiles.d/gluster.conf:2: Failed to resolve user 'gluster': No such process
+/usr/lib/tmpfiles.d/screen.conf:2: Failed to resolve group 'screen': No such process
+Failed to open file "/sys/fs/selinux/checkreqprot": Read-only file system
+
+⏱  Duration: 0s
+manifest example-1.json finished successfully
+tree:  	fa0e466784d49682a1b7b3cb129b7a75b16bff9c2aed6aee3f0f1988056ce85a
+```
+
+The `failed to resolve user` and `group` messages might be different but they can be ignored. The output tells us that the run was successful (`manifest example-1.json finished successfully`). It also shows us the runtime duration for each stage and the output it produces while running it. Stages that call out to other commands usually don't capture the output of those commands unless they have to, and instead display it directly in the build log. Some stages also print their own log output. In many cases the output is useful for tracing and troubleshooting a stage's execution.
```

Comment:
It would be a good idea to also mention the checkreqprot Read-only error which shows up a lot. I'd also say something about how to tell real errors from 'regular' errors like these. Maybe looking for 'finished successfully' is enough, but I'm not sure :)